container-accesslogging/src/main/java/com/yahoo/container/logging/LogFileHandler.java

```diff
@@ -265,24 +265,26 @@ private void internalRotateNow() throws InterruptedException {
         lastRotationTime = now;
         nextRotationTime = 0; //figure it out later (lazy evaluation)
         if ((oldFileName != null)) {
-            if (compressOnRotation) {
-                triggerCompression(oldFileName);
-            } else {
-                NativeIO nativeIO = new NativeIO();
-                nativeIO.dropFileFromCache(new File(oldFileName));
+            File oldFile = new File(oldFileName);
+            if (oldFile.exists()) {
+                if (compressOnRotation) {
+                    triggerCompression(oldFile);
+                } else {
+                    NativeIO nativeIO = new NativeIO();
+                    nativeIO.dropFileFromCache(oldFile);
+                }
             }
         }
     }
 
-    private void triggerCompression(String oldFileName) {
+    private void triggerCompression(File oldFile) {
         try {
-            String gzippedFileName = oldFileName + ".gz";
+            String gzippedFileName = oldFile.getName() + ".gz";
```

Comment:
getName() just returns "name" from "/a/path/to/a/file/name". Is that OK here or should the full path be used? It wasn't obvious to me if the oldFileName string is a full path or not.

Reply:
I will check.

Reply:
Good catch, it is configurable with current directory as default.
Using getPath instead.